packages/@uppy/core/src/Uppy.ts

```diff
@@ -124,15 +124,15 @@ export interface UppyOptions<M extends Meta, B extends Body> {
   logger?: typeof debugLogger
   debug?: boolean
   restrictions: Restrictions
-  meta?: M
+  meta?: Partial<M>
   onBeforeFileAdded?: (
     currentFile: UppyFile<M, B>,
     files: { [key: string]: UppyFile<M, B> },
   ) => UppyFile<M, B> | boolean | undefined
   onBeforeUpload?: (files: {
     [key: string]: UppyFile<M, B>
   }) => { [key: string]: UppyFile<M, B> } | boolean
-  locale?: Locale
+  locale?: OptionalPluralizeLocale
```

Comment:
Are you sure about this? `defaultLocale` is indeed `OptionalPluralizeLocale` but I don't think this is.

Reply:
we're getting typescript errors if it's set to `Locale`

Reply:
That might be but this option should be `Locale`.